Jenkinsfile

```diff
@@ -132,6 +133,54 @@ pipeline {
          junit 'results/results.xml'
        }
      }
-   }
+    }
+
+    // when env.BRANCH_NAME == 'master', we will update the mesosphere:dcos-ui/latest branch
+    // when env.BRANCH_NAME =~ /(tag-name-regex)/, we will open a release PR against dcos/dcos
+    stage('Upload Build') {
+      when {
+        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ SEMVER_REGEX }
+      }
+
+      steps {
+        withCredentials([
+            string(credentialsId: '3f0dbb48-de33-431f-b91c-2366d2f0e1cf',variable: 'AWS_ACCESS_KEY_ID'),
+            string(credentialsId: 'f585ec9a-3c38-4f67-8bdb-79e5d4761937',variable: 'AWS_SECRET_ACCESS_KEY')
+        ]) {
+          sh "BRANCH_NAME=${env.BRANCH_NAME} ./scripts/ci/upload-build"
```

Comment:
this is `BRANCH_NAME=v1.12.0-rc.1` when called from a tag, isnt it? this will break the `upload-build` script (see my comment below) :(